tests/wordlist/user/test_user_word_list_loader.py

```diff
@@ -0,0 +1,18 @@
+#pylint: skip-file
+
+from wordlist.user.user_word_list_loader import UserWordListLoader
+import pytest
+
+# Test the superclass
+def test_UserWordListLoader():
+    # Create an object
+    uwll = UserWordListLoader()
+    # The superclass should have an internal list
+    assert hasattr(uwll, "list")
+
+# Tests the "get" method of the superclass
+def test_get():
```

Comment:
Related to the other comment, this can just check the property

Reply:
Changed.